samples/crypto/ecdh/README.rst

```diff
@@ -7,31 +7,28 @@ Crypto: ECDH
    :local:
    :depth: 2
 
+The ECDH sample shows how to perform an Elliptic-curve Diffie–Hellman key exchange to allow two parties to obtain a common secret.
+
 Overview
 ********
 
-The ECDH sample shows how to perform an Elliptic-curve Diffie–Hellman key exhange to allow two parties to obtain a common secret.
+The sample follows these steps to perform an ECDH key exchange:
+
+1. First, the sample performs initialization:
 
-The sample peforms an ECDH exchange:
-   #. The Platform Security Architecture (PSA) API is initialized.
-   #. Two ECDH keypairs are generated and imported to the PSA keystore.
-   #. ECDH key echange is performed.
-   #. The keypairs are removed from the PSA keystore.
+   a. The Platform Security Architecture (PSA) API is initialized.
+   #. Two random ECDH key pairs are generated and imported into the PSA crypto keystore.
 
-First, the sample performs initalization:
-   #. The Platform Security Architecture (PSA) API is initialized.
-   #. Two random ECDH keypairs are generated and imported in the PSA crypto keystore
+#. A sample plaintext is signed:
 
-First, the sample signs a sample plaintext:
-   #. Signing is performed.
-   #. The keypair is removed from the PSA keystore.
+   a. Signing is performed.
```

Comment:
ECDH does not include signing. Is this a copy-paste from ECDSA example?

The operations is supposed to be a key exchange (sending the public key to the other end) and then calculating a shared secret on both ends...

Reply:
@Vge0rge  ?

Reply:
That is correct, the signing part needs to be removed completely.